sck.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+platform=cdh6.3.0
+
+function help() {
+  echo "sck.sh [--platform=<platform>] command"
```

Comment:
This help message does not seem to match what's happening: running `./sck.sh --platform=cdh6.3.0` gives me `Unknown option: '--platform=cdh6.3.0'`

Reply:
Sorry, I will fix that.